.travis.yml

```diff
@@ -18,91 +18,22 @@ stages:
 - build
 matrix:
   include:
```

Comment:
i organized these task configs so that they are in the same order as they are displayed in travis ci